modules/andes-core/broker/src/main/java/org/wso2/andes/server/AMQBrokerManagerMBean.java

```diff
@@ -432,4 +434,36 @@ public boolean isStatisticsEnabled()
     {
         return getVirtualHost().isStatisticsEnabled();
     }
+
+    /** Updating exclusive Consumer Value     *
+     * @param queueName name of the queue
+     * @param isExclusiveConsumer exclusive consumer value of the queue
+     * @throws IOException
+     * @throws JMException
+     */
+    @Override
+    public void updateExclusiveConsumerValue(String queueName, boolean isExclusiveConsumer) throws IOException,
+            JMException {
+
+        List<AndesQueue> queueList = null;
+        try {
+            queueList = AndesContext.getInstance().getAndesContextStore().getAllQueuesStored();
+
+            if (queueList != null) {
+                for (AndesQueue andesQueue : queueList) {
+
+                    if (andesQueue.queueName.equals(queueName)) {
+                        andesQueue.isExclusiveConsumer = isExclusiveConsumer;
+
+                        //updating the database
+                        AndesContext.getInstance().getAndesContextStore().updateExclusiveConsumerForQueue(queueName,
+                                isExclusiveConsumer);
+                       }
+                    }
+                }
+
+            }catch (AndesException e) {
+            e.printStackTrace();
```

Comment:
e.printStackTrace is not proper way of handling exceptions. Handle the exception